starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -69,4 +70,20 @@ public void create() {
     Assert.assertEquals(actualInventory.get(0).getProductType(), PRODUCT_TYPE);
     Assert.assertNotEquals(actualInventory.get(0).getId(), ID);
   }
+  
+  /**
+   * Test Delete method.
+   */
+  @Test
+  public void delete() {
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    Inventory added = this.inventoryDAO.create(inventory);
```

Comment:
Typically we only want to test one DAO function per @Test, there is another way we can add an inventory within this test class that doesn't rely on the DAO